features/shared/checkout_previous_branch_after_success/same_current_branch/previous_branch_renamed.feature

```diff
@@ -0,0 +1,11 @@
+Feature: when the previous branch is renamed during a Git Town command, the main branch becomes the new previous branch
```

Comment:
> Feature: renaming the previous branch makes the main branch the new previous branch

Looking at this, this doesn't make much sense. Better would be to make the branch with the new name the new previous branch.